src/main/java/com/epam/ta/reportportal/core/launch/impl/MergeLaunchHandler.java

```diff
@@ -197,11 +201,13 @@ private void validateMergingLaunches(List<Launch> launches, User user, Project p
     /**
      * Update test-items of specified launches with new LaunchID
      */
-    private void updateChildrenOfLaunches(String launchId, Set<String> launches, boolean extendDescription) {
+    private void updateChildrenOfLaunches(String launchId, Set<String> launches,
+                                          boolean extendDescription) {
         List<TestItem> testItems = launches.stream().flatMap(id -> {
             Launch launch = launchRepository.findOne(id);
             return testItemRepository.findByLaunch(launch).stream().map(item -> {
                 item.setLaunchRef(launchId);
+                item.setUniqueId(identifierGenerator.generate(item));
```

Comment:
Potential bug here. Add project name for base64 and think about if id was set by user, or launch name was changed